### ISSUE TITLE
Rename the app and change app id

### ISSUE THREAD

**Issue by A6GibKm** [syncthing-gtk/syncthing-gtk#11]
Given the recent fork it might be a good opportunity to consider renaming the app.  In the GNOME community we are *heavily* encouraging third-party apps to use names (at least on the user-facing side of things) that do not allude trademarks such as GNOME, GTK, or using "g" as a prefix, see https://blogs.gnome.org/tbernard/2019/04/26/naming-your-app/.

On a similar note,  strictly speaking this app is not `gtk-syncthing`, it would be good to change the app id to one that does not conflict with the original one (`me.kozec.syncthingtk`), as these act as global unique identifiers and bad things happens on DBus and gsettings when there are two apps with the same ID.

I am very glad to see that  the great work in this app has been resumed and I will be happy to help with the GTK/Flatpak side of things!

**Comment by hrehfeld** [comment]
Maybe just [syncthing-gui](https://duckduckgo.com/?q=%22syncthing-gui%22&atb=v77-3_q&ia=web)?

**Comment by andrewshadura** [comment]
Yes, I think syncthing-gui is a better name. Back in the day when I was the sole remaining maintainer of Syncthing-GTK, I considered renaming it because Syncthing-GTK seemed a bit wrong (e.g. why would it matter that it’s GTK?), but I never went ahead with it.

**Comment by hrehfeld** [comment]
So, do we recreate the whole org/repo then? Or is there a better way?

Ah:
1. rename this org: https://docs.github.com/en/organizations/managing-organization-settings/renaming-an-organization
2. recreate `syncthing-gtk` org to block hijacking the name (?)
3. non-fork-push the current repo to `syncthing-gui/syncthing-gui `
4. let the old `syncthing-gtk` repo just sit there with a readme change?

Any objections?

**Comment by imsodin** [comment]
Just my 2cents: A more specific name would be nice. As the linked search shows, "syncthing GUI" is often associated with the web UI bundled with syncthing. And then there's plenty of other projects that are somehow UIs for syncthing, though admittedly most are more system integrations while syncthing-gtk is actually a full-fledged UI (and a system integration). "syncthing-gui" is bound to be confusing for users in search and support. Basically anything original will do. I am clearly terrible at naming, because the first thing I came up with is "syncguithing" :P And more thinking doesn't produce much better stuff (not using dashes/upper-case, insert at liberty except at/before the "thing"):
 - syncfront
 - syncthingfront
 - visualsyncthing (please don't judge me, I resolved to write down everything that came to my mind before this came to my mind)
 - syncthing-desktop
 - syncthingdisplay

**Comment by mrmeszaros** [comment]
@imsodin I agree with the notion that more specific is better - as it will help users (such as me) identify the item.

However, I think that `syncthing-gtk` is a pretty good name, and I have four points:

* it has **syncthing** in it's name - this links to it's main purpose (being a syncthing app)
* the name is **familiar** - the same name as it's previous incarnation (with it's previous maintainer, kozec)
* it has **GTK** in its name - this denotes to me that _GTK-themes_ can be applied to it, and Qt/KDE themes _not_.
* the name is **descriptive** - I assume that this will _not be rewritten_ in any other GUI framework in the near future

That all said, I propose my original name ideas - uncensored (sorry in advance):
SyncHound, SyncBlob, SyncThingamabob, SyncThingo, SyncTheThing

**Comment by benjamin051000** [comment]
The thing I like about this app compared to others is that there is no "weird" web UI. It's all _contained_. This makes it more user-friendly. Therefore, maybe it could be something like "SyncEasy" or "SyncSimple," something that conveys it is more user-friendly or easier to use or simpler than other SyncThing clients.

**Comment by acolomb** [comment]
* SyncHound, SyncBlob - Do not have Syncthing in their name, which I agree would be a step backward from Syncthing-GTK. 
* SyncThingamabob - I don't like the fact that it's all one word, and a pretty unusual parts. Plus the capitalization should never be SyncThing, as that's explicitly not what the upstream project calls it. 
* SyncThingo - too close, looks like just a typo and will be treated as such by search engines. 
* SyncTheThing - I find that more confusing because it looks like an alternative, not an extension / related app to Syncthing. 

My 2 cents on those... Frankly I have no issue with keeping the Syncthing-GTK legacy.

**Comment by frandavid100** [comment]
> Maybe it could be something like "SyncEasy" or "SyncSimple," something that conveys it is more user-friendly or easier to use or simpler than other SyncThing clients.

How about "Simple Sync"?

**Comment by Salamandar** [comment]
> Maybe just [syncthing-gui](https://duckduckgo.com/?q=%22syncthing-gui%22&atb=v77-3_q&ia=web)?

I kinda like it, but not 100%. What about Syncthingui ?

**Comment by mrmeszaros** [comment]
If I understand correctly this application is for GNOME and other GTK desktops.

So it might be good to follow the [GNOME Human Interface Guidelines on App Naming](https://developer.gnome.org/hig/guidelines/app-naming.html).

Based on that, the closest option would be **Syncthing Desktop** (`syncthing-desktop`) in my opinion.

But, as the document states, we could brainstorm in properly.

**Comment by Salamandar** [comment]
I don’t really like "Syncthing Desktop" but that might be just a wrong first impression. If other people like it, that’s fine with me.

**Comment by mrmeszaros** [comment]
Maybe it sounds a bit generic - and that throws You off?
Anyway, it is open to suggestions.

I think Syncthing Desktop would be better than the current one - regarding GNOME app naming.

As for the DBus id... IDK

**Comment by Salamandar** [comment]
Yeah that might be the "genericness" that bugs me off.
But for sure it would still be better than the current name.

**Comment by A6GibKm** [comment]
The advice we give in the gnome community is to have unique names not related to other (possibly trademarked) projects, e.g. not prefixing with g/gtk or in this case 'Syncthing'.

Ofc this is up to you, but it is worth to have in mind

**Comment by Salamandar** [comment]
Well I can understand in some cases, but in others like this one i disagree : people search for a syncthing client specifically, not "some synchro software".

**Comment by A6GibKm** [comment]
I mean sure, just have in mind that these names might be trademarked (now or in the future) and it might be illegal to use their name in your software, even if not the case it is still rude to just take advantage of the brand of another project.

When you have third-party projects, whatever problem users encounter with the software, they will irremediably  associate it with a problem with the upstream software itself, even if the fault lies in the third-party developer.

**Comment by Salamandar** [comment]
Yeah, indeed. This might be a nice idea to ask the Syncthing guys if "synchting desktop" is OK with them.

**Comment by acolomb** [comment]
Speaking as one of the upstream developers, but I'm not responsible for branding or trademarks.

I'd be fine with `Syncthing Desktop` (without the typos).  In this case I think it is even mandatory to include `Syncthing` in the app name, because it is just a wrapper, targeted exactly for that upstream project.  Anything even more generic would fail to express that connection.

**Comment by Salamandar** [comment]
Thanks @acolomb <3 I tried contacting your friend @imsodin on Matrix but no reply yet :)
 
>  In this case I think it is even mandatory to include Syncthing in the app name, because it is just a wrapper, targeted exactly for that upstream project. Anything even more generic would fail to express that connection.

Yeah, agreed ! Maybe if someday we want to support other sync protocols that might change but for now, it's really syncthing-related :D

**Comment by imsodin** [comment]
I don't think there's anything mandatory about including "syncthing". Making it very clear to users that you are running/wrapping syncthing would be nice and imo also good for your users - having it somehow in the name is one way of doing that, but not the only. Generally regarding naming I stand by my earlier point: If you rename, I'd advice to use something unique, not syncthing-gui - that's too generic for my taste and syncthing's own web UI is often referred to it's GUI. And syncthing-gui is about as close to that (and official/native looking) as it gets.

(I need to finally setup some matrix bridges, using a gazillion IMs is annoying and makes me miss such messages - I'll check it more frequently now but can't guarantee any timeframe for that attention as I don't have any active comms there atm :) ).

**Comment by Rudd-O** [comment]
Why not _Syncontrol_?

**Comment by acolomb** [comment]
There may be trademarks on the name _Syncontrol_ though. A Chinese quality assurance software company and some electrical desk adjustment controller come up on a quick Google search.

**Comment by Rudd-O** [comment]
Syncontroller?

**Comment by mikoducky** [comment]
Syncthing Hub

Because it comes from Flathub. And because it is the new desktop hub version that will be cross-platform for all flatpak installs.

**Comment by Rudd-O** [comment]
Yeah I like Syncthing Hub.  Sync Hub would also be acceptable.

**Comment by acolomb** [comment]
"Hub" somehow implies for me that it has to do with several Syncthings. Which is okay, as that would actually be a great feature addition - being able to switch between / handle multiple API connections. So I wouldn't object.

**Comment by tehbuhtig** [comment]
any decisions made here? there are still folks filing [issues](https://github.com/kozec/syncthing-gtk/issues/603) against the new flatpak on the old github page...
